.nuget/V3/packages.config

```diff
@@ -1,7 +1,3 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
```

Comment:
We could, but i'd leave it here as a placeholder for V3 packages, should we need some.